Atomic/atomic.py

```diff
@@ -711,7 +709,27 @@ def get_col_lengths(_images):
                                               "CREATED",
                                               "VIRTUAL SIZE",
                                               "TYPE"))
+
+            for image in image_data:
```

Comment:
we should change it so that we can iterate over `_images` instead of having `image_data`. 

It will also simplify the `get_col_lengths` function, that can do: `repo_tags = [[i["repo"], i["tag"]] for i in _images]` and get rid completely of `split_repo_tags`.